HACKING.md

```diff
@@ -299,6 +299,22 @@ def some_task(self, file_id):
 
 In this example the producer side REST endpoint creates a token with the scope of the current user, then passes the REST argument `file_id` and the token to the task object's `.delay(...)` function. On the consumer side, the task has access to the GirderClient instance through `self.girder_client`. To get access to `self` the task must be decorated with `@app.task(bind=True)` (See [bound tasks](http://docs.celeryproject.org/en/latest/userguide/tasks.html#bound-tasks) for more information).
 
+##### What is Girder is served over an HTTPS connection ?
```

Comment:
```suggestion
##### Customizing communication back to Girder server
```

I'm OK with the code changes you've added, but I think we should document it more generally as a feature, i.e. configurability of the request session. `verify: False` could still be used as the example.

Aside from that, it's more accurate to say "What if Girder is served over an _unauthenticated_ TLS connection?" which is kind of a strange use case. What's the reason you need this, out of curiosity?

Reply:
Hi Zach, thanks for the review.

I will document it more tomorrow.
We need this for a project where Girder is not the front-ent webserver (an Apache with load-balancer is set up in front in another VM). However, the security consultants hired by the client require us to cypher the communications between Apache and Girder. To do that, a self-signed certificate is created and we don't want to bother with signing it.

Reply:
That's an... interesting... threat model. Lots of people use tools like [Consul](https://www.consul.io/) for secure inter-service authentication. I'd suggest checking that out if security is something you care about. It manages the hassles of CAs and TLS for you pretty nicely.

Reply:
Thanks for the suggestion, I'll be sure to suggest it to the client.

I changed my `HACKING.md` section to a feature documentation. Is that what you had in mind ?